### DISCUSSION TITLE
不支持给 其他设备发消息？

### DISCUSSION THREAD

**Discussion by huiqiangyang** [mamoe/mirai#1350] [Q&A]
![image](https://user-images.githubusercontent.com/13185035/122642648-17a81100-d13e-11eb-9166-b3619f649cbe.png)

![image](https://user-images.githubusercontent.com/13185035/122642654-1ecf1f00-d13e-11eb-82e6-25fa16cbfbcb.png)

## 版本
mirai-core-jvm-2.6.5.jar

**Comment by Him188** [comment]
不支持
